### ISSUE TITLE
Git-Bash shortcut not opening Git-Bash console

### ISSUE THREAD

**Issue by benjamin-sandham** [git-for-windows/git#2597]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?
64-bit
```
$ git --version --build-options

** git version 2.26.2.windows.1
cpu: x86_64
built from commit: 4267c88d0d8936ae723bba55e0576b1c4f3d4e1a
sizeof-long: 4
sizeof-size_t: 8**
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?
Windows 10 64-bit
```
$ cmd.exe /c ver

**Microsoft Windows [Version 10.0.18363.778]**
```

 - All defaults except for:
     Used Windows’ default console window option
     Followed: https://www.earthdatascience.org/workshops/setup-earth-analytics-python/setup-git- 
     bash-conda/
     To be able to use conda and git commands from same console.

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

**Editor Option: VIM
Custom Editor Path:
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: ConHost
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Disabled**
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

**Not that I am aware. **

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

** Bash **

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
** No commands. I closed the bash console window, and the next time I tried opening one the shortcut would not open a console window.

- Application shortcut targets: "C:\Program Files\Git\git-bash.exe" --cd-to-home
- But I can navigate to  C:\Program Files\Git\bin\bash.exe which will open a console

I can change where the application target is, but I just want to ensure that my setup is not causing issues down the line. **
```
 - What did you expect to occur after running these commands?

** N/A **

 - What actually happened instead?

** N/A **

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

** N/A **

**Comment by PhilipOakley** [comment]
Wild quick guess: do check the release notes regarding `winpty` and terminal windows https://github.com/git-for-windows/build-extra/blob/master/ReleaseNotes.md#known-issues, given that link contains "setup-earth-analytics-python". (I haven't looked at those instructions)
 
Also check dscho's clarification at https://github.com/git-for-windows/git/issues/2594#issuecomment-616677790 as I maybe way off the mark.

**Comment by rimrul** [comment]
@PhilipOakley , I think you're a little off the mark here. They said they use the "Use Windows’ default console window" option, so the `winpty` things don't apply. They basically already did what dscho suggests in the comment you linked. The `winpty` issue would also manifest in a different way. The Git-Bash window would show up and issues would only appear after calling `python` from within that window. This is something else.

@benjamin-sandham do I read this part correctly?

>``` 
>C:\Program Files\Git>git-bash.exe
>```
>Launches git console.

Launching `git-bash.exe` from within the `C:\Program Files\Git` folder using `cmd.exe` does show the Git bash window, but clicking the shortcut doesn't?

>But I can navigate to C:\Program Files\Git\bin\bash.exe which will open a console

Yeah, that isn't quite the same as `git-bash.exe`.

>Application shortcut targets: "C:\Program Files\Git\git-bash.exe" --cd-to-home

That does sound correct. That's exactly what my working Git Bash shortcut targets on multiple systems. It says 

>Start in: %HOMEDRIVE%%HOMEPATH%
>Shortcut key: None
>Run: Normal window

below that. The shortcut file also has "Read" and "Read, execute" NTFS permissions granted to "Users" and "Everyone".

Could you check the permissions on your shortcut and try the following in `cmd.exe`?

```
%HOMEDRIVE%
cd %HOMEDRIVE%%HOMEPATH%
C:\Program Files\Git\git-bash.exe
```

**Comment by benjamin-sandham** [comment]
@rimrul 
Navigating to 
`%HOMEDRIVE%
cd %HOMEDRIVE%%HOMEPATH%`
Gives path
`C:\Users\bsand>`
And executing 
`"C:\Program Files\Git\git-bash.exe"`

Opens a bash console window. I had to use quotation marks as the following was occurring
`C:\Users\bsand>C:\Program Files\Git\git-bash.exe
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.`

My shortcut properties display the following:
`Start in: %HOMEDRIVE%%HOMEPATH%
Shortcut key: None
Run: Normal window`

My shortcut file also has "Read" and "Read, execute" NTFS permissions granted to "Users" and "Everyone".

**Comment by rimrul** [comment]
Yeah, the missing quotation marks where my mistake. That looks good, too. sounds like an issue with `--cd-to-home`. could you check `echo %HOME%`?

**Comment by benjamin-sandham** [comment]
```
C:\Users\bsand>echo %HOME%
%HOME%
```

Would you be able to explain the difference between bash.exe and git-bash.exe? I can set the application shortcut file to "C:\Program Files\Git\bin\bash.exe" which will then allow me to click the icon and open a console window, but I don't fundamentally understand the difference between the two.

**Comment by rafrafek** [comment]
It looks similar to "Open Windows Terminal here" issue with Windows Terminal:
https://github.com/microsoft/terminal/issues/8933#issuecomment-829289887

I have also experienced weird behavior when opening Git Bash by "Git Bash Here" (with right mouse click inside directory) - repository configuration was not being loaded.
